ui/README.adoc

```diff
@@ -0,0 +1,17 @@
+= Mobile Security Service UI
```

Comment:
I'd like to suggest to add on top of this file the following code. It will keep the same style from the other readmes. 

```
ifdef::env-github[]
:status:
:outfilesuffix: .adoc
:caution-caption: :fire:
:important-caption: :exclamation:
:note-caption: :paperclip:
:tip-caption: :bulb:
:warning-caption: :warning:
:table-caption!:
endif::[]

:toc:
:toc-placement!:
```

Reply:
Will do @camilamacedo86 I didn't include the toc as it was only one entry and the README is very short. I don't think it's necessary as I believe this README is going to be very short anyway as I don't want to duplicate content from the root README